services/QuillLMS/client/app/bundles/Teacher/components/accounts/edit/teacher_linked_accounts.jsx

```diff
@@ -97,11 +112,35 @@ export default class TeacherLinkedAccounts extends React.Component {
 
   renderClasslinkSection = () => {
     let actionElement, copy
+
+    const { user } = this.props
+    copy = 'ClassLink is not linked'
+
     if (this.isLinkedToClasslink()) {
       copy = 'ClassLink is linked'
+      if (!hasSchoolOrDistrictPremium(user)) {
+        actionElement = (
+          <button
+            className='google-or-clever-action focus-on-light'
```

Comment:
Maybe we should rename this class `provider-class`, in keeping with your other changes?